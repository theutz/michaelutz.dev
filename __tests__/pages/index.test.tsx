import { render } from "@testing-library/react";
import Index from "../../pages/index";

describe("<Index />", () => {
  it("renders", () => {
    // Arrange

    // Act
    const { asFragment } = render(<Index />);

    // Assert
    expect(asFragment()).toMatchSnapshot();
  });

  it("breaks", () => {
    expect(true).toBe(false);
  });
});
