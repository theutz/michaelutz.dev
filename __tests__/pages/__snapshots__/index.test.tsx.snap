// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index /> renders 1`] = `
<DocumentFragment>
  <div
    class="relative"
  >
    <div
      class="absolute inset-x-0 bottom-0 bg-gray-100 h-1/2"
    />
    <div
      class="mx-auto max-w-7xl sm:px-6 lg:px-8"
    >
      <div
        class="relative shadow-xl sm:rounded-2xl sm:overflow-hidden"
      >
        <div
          class="absolute inset-0"
        >
          <img
            alt="People working on laptops"
            class="object-cover w-full h-full"
            src="https://images.unsplash.com/photo-1521737852567-6949f3f9f2b5?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2830&q=80&sat=-100"
          />
          <div
            class="absolute inset-0 bg-gradient-to-r from-purple-800 to-indigo-700 mix-blend-multiply"
          />
        </div>
        <div
          class="relative px-4 py-16 sm:px-6 sm:py-24 lg:py-32 lg:px-8"
        >
          <h1
            class="text-4xl font-extrabold tracking-tight text-center sm:text-5xl lg:text-6xl"
          >
            <span
              class="block text-white"
            >
              Not Your Average
            </span>
            <span
              class="block text-indigo-200"
            >
              Web Dude
            </span>
          </h1>
          <p
            class="max-w-lg mx-auto mt-6 text-xl text-center text-indigo-200 sm:max-w-3xl"
          >
            I'm passionate about creating experiences for the web that are useful, enjoyable, and intuitive.
          </p>
          <div
            class="max-w-sm mx-auto mt-10 sm:max-w-none sm:flex sm:justify-center"
          >
            <div
              class="space-y-4 sm:space-y-0 sm:mx-auto sm:inline-grid sm:grid-cols-2 sm:gap-5"
            >
              <a
                class="flex items-center justify-center px-4 py-3 text-base font-medium text-indigo-700 bg-white border border-transparent rounded-md shadow-sm hover:bg-indigo-50 sm:px-8"
                href="#"
              >
                Learn More
              </a>
              <a
                class="flex items-center justify-center px-4 py-3 text-base font-medium text-white bg-indigo-500 border border-transparent rounded-md shadow-sm bg-opacity-60 hover:bg-opacity-70 sm:px-8"
                href="#"
              >
                Reach Out
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="bg-gray-100"
    data-cy="logo-cloud"
  >
    <div
      class="px-4 py-16 mx-auto max-w-7xl sm:px-6 lg:px-8"
    >
      <p
        class="text-sm font-semibold tracking-wide text-center text-gray-500 uppercase"
      >
        Trusted by a handful of former employers
      </p>
      <div
        class="mt-6 grid grid-cols-2 gap-8 md:grid-cols-6 lg:grid-cols-5"
        data-cy="logo-cloud-container"
      >
        <div
          class="relative filter grayscale opacity-40 h-12 md:col-span-2 flex justify-center lg:col-span-1 col-span-1"
        >
          <div
            style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
          >
            <noscript />
            <img
              alt="Unity"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
          </div>
        </div>
        <div
          class="relative filter grayscale opacity-40 h-12 md:col-span-2 flex justify-center lg:col-span-1 col-span-1"
        >
          <div
            style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
          >
            <noscript />
            <img
              alt="Smartly.io"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
          </div>
        </div>
        <div
          class="relative filter grayscale opacity-40 h-12 md:col-span-2 flex justify-center lg:col-span-1 col-span-1"
        >
          <div
            style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
          >
            <noscript />
            <img
              alt="Quentic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
          </div>
        </div>
        <div
          class="relative filter grayscale opacity-40 h-12 md:col-span-2 flex justify-center lg:col-span-1 col-span-1 md:col-start-2"
        >
          <div
            style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
          >
            <noscript />
            <img
              alt="Universal Hospital Systems"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
          </div>
        </div>
        <div
          class="relative filter grayscale opacity-40 h-12 md:col-span-2 flex justify-center lg:col-span-1 col-span-2 md:col-start-4"
        >
          <div
            style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
          >
            <noscript />
            <img
              alt="Emergency Reporting"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="relative pt-16 pb-32 overflow-hidden"
  >
    <div
      aria-hidden="true"
      class="absolute inset-x-0 top-0 h-48 bg-gradient-to-b from-gray-100"
    />
    <div
      class="relative"
    >
      <div
        class="lg:mx-auto lg:max-w-7xl lg:px-8 lg:grid lg:grid-cols-2 lg:grid-flow-col-dense lg:gap-24"
      >
        <div
          class="max-w-xl px-4 mx-auto sm:px-6 lg:py-16 lg:max-w-none lg:mx-0 lg:px-0"
        >
          <div>
            <div>
              <span
                class="flex items-center justify-center w-12 h-12 rounded-md bg-gradient-to-r from-purple-600 to-indigo-600"
              >
                <svg
                  aria-hidden="true"
                  class="w-6 h-6 text-white"
                  fill="none"
                  stroke="currentColor"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M14 10h4.764a2 2 0 011.789 2.894l-3.5 7A2 2 0 0115.263 21h-4.017c-.163 0-.326-.02-.485-.06L7 20m7-10V5a2 2 0 00-2-2h-.095c-.5 0-.905.405-.905.905 0 .714-.211 1.412-.608 2.006L7 11v9m7-10h-2M7 20H5a2 2 0 01-2-2v-6a2 2 0 012-2h2.5"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="2"
                  />
                </svg>
              </span>
            </div>
            <div
              class="mt-6"
            >
              <h2
                class="text-3xl font-extrabold tracking-tight text-gray-900"
              >
                The Humans Have Spoken
              </h2>
              <p
                class="mt-4 text-lg text-gray-500"
              >
                Building tech products is also about building relationships. Here's what some of my co-workers have to say about working with me.
              </p>
              <div
                class="mt-6"
              >
                <button
                  class="flex inline-flex px-4 py-2 text-base font-medium text-white border border-transparent bg-gradient-to-r from-purple-600 to-indigo-600 bg-origin-border rounded-md shadow-sm hover:from-purple-700 hover:to-indigo-700"
                >
                  Read Another
                </button>
              </div>
            </div>
          </div>
          <div
            class="pt-6 mt-8 border-t border-gray-200"
          >
            <blockquote>
              <div>
                <p
                  class="text-base text-gray-500 whitespace-pre-line"
                >
                  “I worked alongside Michael on the Web Development Team at FRN/UHS. Mike is super friendly and top of the line at his craft. His work in DevOps and JavaScript elevated our team's workflow to the next level, making everyone on our team better at what they do. Mike was our "go-to" guy whenever we had complex development issues. He always communicated clearly and timely while consistently delivering excellent results. It was a privilege to learn from and work with Mike!”
                </p>
              </div>
              <footer
                class="mt-3"
              >
                <div
                  class="flex items-center space-x-3"
                >
                  <div
                    class="relative flex-shrink-0"
                  >
                    <div
                      class="w-6 h-6"
                    >
                      <div
                        style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
                      >
                        <noscript />
                        <img
                          alt="Benjamin Wright"
                          class="rounded-full"
                          decoding="async"
                          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
                        />
                      </div>
                    </div>
                  </div>
                  <div
                    class="text-base font-medium text-gray-700"
                  >
                    Benjamin Wright, Director of Ecommerce @ Space Race
                  </div>
                </div>
              </footer>
            </blockquote>
          </div>
        </div>
        <div
          class="mt-12 sm:mt-16 lg:mt-0"
        >
          <div
            class="pl-4 -mr-48 sm:pl-6 md:-mr-16 lg:px-0 lg:m-0 lg:relative lg:h-full"
          >
            <img
              alt="Inbox user interface"
              class="w-full shadow-xl rounded-xl ring-1 ring-black ring-opacity-5 lg:absolute lg:left-0 lg:h-full lg:w-auto lg:max-w-none"
              src="https://tailwindui.com/img/component-images/inbox-app-screenshot-1.jpg"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      class="mt-24"
    >
      <div
        class="lg:mx-auto lg:max-w-7xl lg:px-8 lg:grid lg:grid-cols-2 lg:grid-flow-col-dense lg:gap-24"
      >
        <div
          class="max-w-xl px-4 mx-auto sm:px-6 lg:py-32 lg:max-w-none lg:mx-0 lg:px-0 lg:col-start-2"
        >
          <div>
            <div>
              <span
                class="flex items-center justify-center w-12 h-12 rounded-md bg-gradient-to-r from-purple-600 to-indigo-600"
              >
                <svg
                  aria-hidden="true"
                  class="w-6 h-6 text-white"
                  fill="none"
                  stroke="currentColor"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M5 3v4M3 5h4M6 17v4m-2-2h4m5-16l2.286 6.857L21 12l-5.714 2.143L13 21l-2.286-6.857L5 12l5.714-2.143L13 3z"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="2"
                  />
                </svg>
              </span>
            </div>
            <div
              class="mt-6"
            >
              <h2
                class="text-3xl font-extrabold tracking-tight text-gray-900"
              >
                Better understand your customers
              </h2>
              <p
                class="mt-4 text-lg text-gray-500"
              >
                Semper curabitur ullamcorper posuere nunc sed. Ornare iaculis bibendum malesuada faucibus lacinia porttitor. Pulvinar laoreet sagittis viverra duis. In venenatis sem arcu pretium pharetra at. Lectus viverra dui tellus ornare pharetra.
              </p>
              <div
                class="mt-6"
              >
                <a
                  class="inline-flex px-4 py-2 text-base font-medium text-white border border-transparent bg-gradient-to-r from-purple-600 to-indigo-600 bg-origin-border rounded-md shadow-sm hover:from-purple-700 hover:to-indigo-700"
                  href="#"
                >
                  Get started
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="mt-12 sm:mt-16 lg:mt-0 lg:col-start-1"
        >
          <div
            class="pr-4 -ml-48 sm:pr-6 md:-ml-16 lg:px-0 lg:m-0 lg:relative lg:h-full"
          >
            <img
              alt="Customer profile user interface"
              class="w-full shadow-xl rounded-xl ring-1 ring-black ring-opacity-5 lg:absolute lg:right-0 lg:h-full lg:w-auto lg:max-w-none"
              src="https://tailwindui.com/img/component-images/inbox-app-screenshot-2.jpg"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="bg-gradient-to-r from-purple-800 to-indigo-700"
  >
    <div
      class="max-w-4xl px-4 py-16 mx-auto sm:px-6 sm:pt-20 sm:pb-24 lg:max-w-7xl lg:pt-24 lg:px-8"
    >
      <h2
        class="text-3xl font-extrabold tracking-tight text-white"
      >
        Inbox support built for efficiency
      </h2>
      <p
        class="max-w-3xl mt-4 text-lg text-purple-200"
      >
        Ac tincidunt sapien vehicula erat auctor pellentesque rhoncus. Et magna sit morbi lobortis. Blandit aliquam sit nisl euismod mattis in.
      </p>
      <div
        class="mt-12 grid grid-cols-1 gap-x-6 gap-y-12 sm:grid-cols-2 lg:mt-16 lg:grid-cols-4 lg:gap-x-8 lg:gap-y-16"
      >
        <div>
          <div>
            <span
              class="flex items-center justify-center w-12 h-12 bg-white rounded-md bg-opacity-10"
            >
              <svg
                aria-hidden="true"
                class="w-6 h-6 text-white"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M20 13V6a2 2 0 00-2-2H6a2 2 0 00-2 2v7m16 0v5a2 2 0 01-2 2H6a2 2 0 01-2-2v-5m16 0h-2.586a1 1 0 00-.707.293l-2.414 2.414a1 1 0 01-.707.293h-3.172a1 1 0 01-.707-.293l-2.414-2.414A1 1 0 006.586 13H4"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
            </span>
          </div>
          <div
            class="mt-6"
          >
            <h3
              class="text-lg font-medium text-white"
            >
              Unlimited Inboxes
            </h3>
            <p
              class="mt-2 text-base text-purple-200"
            >
              Ac tincidunt sapien vehicula erat auctor pellentesque rhoncus. Et magna sit morbi lobortis.
            </p>
          </div>
        </div>
        <div>
          <div>
            <span
              class="flex items-center justify-center w-12 h-12 bg-white rounded-md bg-opacity-10"
            >
              <svg
                aria-hidden="true"
                class="w-6 h-6 text-white"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M12 4.354a4 4 0 110 5.292M15 21H3v-1a6 6 0 0112 0v1zm0 0h6v-1a6 6 0 00-9-5.197M13 7a4 4 0 11-8 0 4 4 0 018 0z"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
            </span>
          </div>
          <div
            class="mt-6"
          >
            <h3
              class="text-lg font-medium text-white"
            >
              Manage Team Members
            </h3>
            <p
              class="mt-2 text-base text-purple-200"
            >
              Ac tincidunt sapien vehicula erat auctor pellentesque rhoncus. Et magna sit morbi lobortis.
            </p>
          </div>
        </div>
        <div>
          <div>
            <span
              class="flex items-center justify-center w-12 h-12 bg-white rounded-md bg-opacity-10"
            >
              <svg
                aria-hidden="true"
                class="w-6 h-6 text-white"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M19 7l-.867 12.142A2 2 0 0116.138 21H7.862a2 2 0 01-1.995-1.858L5 7m5 4v6m4-6v6m1-10V4a1 1 0 00-1-1h-4a1 1 0 00-1 1v3M4 7h16"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
            </span>
          </div>
          <div
            class="mt-6"
          >
            <h3
              class="text-lg font-medium text-white"
            >
              Spam Report
            </h3>
            <p
              class="mt-2 text-base text-purple-200"
            >
              Ac tincidunt sapien vehicula erat auctor pellentesque rhoncus. Et magna sit morbi lobortis.
            </p>
          </div>
        </div>
        <div>
          <div>
            <span
              class="flex items-center justify-center w-12 h-12 bg-white rounded-md bg-opacity-10"
            >
              <svg
                aria-hidden="true"
                class="w-6 h-6 text-white"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M11 5H6a2 2 0 00-2 2v11a2 2 0 002 2h11a2 2 0 002-2v-5m-1.414-9.414a2 2 0 112.828 2.828L11.828 15H9v-2.828l8.586-8.586z"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
            </span>
          </div>
          <div
            class="mt-6"
          >
            <h3
              class="text-lg font-medium text-white"
            >
              Compose in Markdown
            </h3>
            <p
              class="mt-2 text-base text-purple-200"
            >
              Ac tincidunt sapien vehicula erat auctor pellentesque rhoncus. Et magna sit morbi lobortis.
            </p>
          </div>
        </div>
        <div>
          <div>
            <span
              class="flex items-center justify-center w-12 h-12 bg-white rounded-md bg-opacity-10"
            >
              <svg
                aria-hidden="true"
                class="w-6 h-6 text-white"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M9 17v-2m3 2v-4m3 4v-6m2 10H7a2 2 0 01-2-2V5a2 2 0 012-2h5.586a1 1 0 01.707.293l5.414 5.414a1 1 0 01.293.707V19a2 2 0 01-2 2z"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
            </span>
          </div>
          <div
            class="mt-6"
          >
            <h3
              class="text-lg font-medium text-white"
            >
              Team Reporting
            </h3>
            <p
              class="mt-2 text-base text-purple-200"
            >
              Ac tincidunt sapien vehicula erat auctor pellentesque rhoncus. Et magna sit morbi lobortis.
            </p>
          </div>
        </div>
        <div>
          <div>
            <span
              class="flex items-center justify-center w-12 h-12 bg-white rounded-md bg-opacity-10"
            >
              <svg
                aria-hidden="true"
                class="w-6 h-6 text-white"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M3 10h10a8 8 0 018 8v2M3 10l6 6m-6-6l6-6"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
            </span>
          </div>
          <div
            class="mt-6"
          >
            <h3
              class="text-lg font-medium text-white"
            >
              Saved Replies
            </h3>
            <p
              class="mt-2 text-base text-purple-200"
            >
              Ac tincidunt sapien vehicula erat auctor pellentesque rhoncus. Et magna sit morbi lobortis.
            </p>
          </div>
        </div>
        <div>
          <div>
            <span
              class="flex items-center justify-center w-12 h-12 bg-white rounded-md bg-opacity-10"
            >
              <svg
                aria-hidden="true"
                class="w-6 h-6 text-white"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M8 10h.01M12 10h.01M16 10h.01M9 16H5a2 2 0 01-2-2V6a2 2 0 012-2h14a2 2 0 012 2v8a2 2 0 01-2 2h-5l-5 5v-5z"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
            </span>
          </div>
          <div
            class="mt-6"
          >
            <h3
              class="text-lg font-medium text-white"
            >
              Email Commenting
            </h3>
            <p
              class="mt-2 text-base text-purple-200"
            >
              Ac tincidunt sapien vehicula erat auctor pellentesque rhoncus. Et magna sit morbi lobortis.
            </p>
          </div>
        </div>
        <div>
          <div>
            <span
              class="flex items-center justify-center w-12 h-12 bg-white rounded-md bg-opacity-10"
            >
              <svg
                aria-hidden="true"
                class="w-6 h-6 text-white"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M4.318 6.318a4.5 4.5 0 000 6.364L12 20.364l7.682-7.682a4.5 4.5 0 00-6.364-6.364L12 7.636l-1.318-1.318a4.5 4.5 0 00-6.364 0z"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
            </span>
          </div>
          <div
            class="mt-6"
          >
            <h3
              class="text-lg font-medium text-white"
            >
              Connect with Customers
            </h3>
            <p
              class="mt-2 text-base text-purple-200"
            >
              Ac tincidunt sapien vehicula erat auctor pellentesque rhoncus. Et magna sit morbi lobortis.
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="relative bg-gray-900"
  >
    <div
      class="absolute inset-x-0 bottom-0 h-80 xl:top-0 xl:h-full"
    >
      <div
        class="w-full h-full xl:grid xl:grid-cols-2"
      >
        <div
          class="h-full xl:relative xl:col-start-2"
        >
          <img
            alt="People working on laptops"
            class="object-cover w-full h-full opacity-25 xl:absolute xl:inset-0"
            src="https://images.unsplash.com/photo-1521737852567-6949f3f9f2b5?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2830&q=80&sat=-100"
          />
          <div
            aria-hidden="true"
            class="absolute inset-x-0 top-0 h-32 bg-gradient-to-b from-gray-900 xl:inset-y-0 xl:left-0 xl:h-full xl:w-32 xl:bg-gradient-to-r"
          />
        </div>
      </div>
    </div>
    <div
      class="max-w-4xl px-4 mx-auto sm:px-6 lg:max-w-7xl lg:px-8 xl:grid xl:grid-cols-2 xl:grid-flow-col-dense xl:gap-x-8"
    >
      <div
        class="relative pt-12 pb-64 sm:pt-24 sm:pb-64 xl:col-start-1 xl:pb-24"
      >
        <h2
          class="text-sm font-semibold tracking-wide uppercase"
        >
          <span
            class="text-transparent bg-gradient-to-r from-purple-300 to-indigo-300 bg-clip-text"
          >
            Valuable Metrics
          </span>
        </h2>
        <p
          class="mt-3 text-3xl font-extrabold text-white"
        >
          Get actionable data that will help grow your business
        </p>
        <p
          class="mt-5 text-lg text-gray-300"
        >
          Rhoncus sagittis risus arcu erat lectus bibendum. Ut in adipiscing quis in viverra tristique sem. Ornare feugiat viverra eleifend fusce orci in quis amet. Sit in et vitae tortor, massa. Dapibus laoreet amet lacus nibh integer quis. Eu vulputate diam sit tellus quis at.
        </p>
        <div
          class="mt-12 grid grid-cols-1 gap-y-12 gap-x-6 sm:grid-cols-2"
        >
          <p>
            <span
              class="block text-2xl font-bold text-white"
            >
              8K+
            </span>
            <span
              class="block mt-1 text-base text-gray-300"
            >
              <span
                class="font-medium text-white"
              >
                Companies
              </span>
               use laoreet amet lacus nibh integer quis.
            </span>
          </p>
          <p>
            <span
              class="block text-2xl font-bold text-white"
            >
              25K+
            </span>
            <span
              class="block mt-1 text-base text-gray-300"
            >
              <span
                class="font-medium text-white"
              >
                Countries around the globe
              </span>
               lacus nibh integer quis.
            </span>
          </p>
          <p>
            <span
              class="block text-2xl font-bold text-white"
            >
              98%
            </span>
            <span
              class="block mt-1 text-base text-gray-300"
            >
              <span
                class="font-medium text-white"
              >
                Customer satisfaction
              </span>
               laoreet amet lacus nibh integer quis.
            </span>
          </p>
          <p>
            <span
              class="block text-2xl font-bold text-white"
            >
              12M+
            </span>
            <span
              class="block mt-1 text-base text-gray-300"
            >
              <span
                class="font-medium text-white"
              >
                Issues resolved
              </span>
               lacus nibh integer quis.
            </span>
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    class="bg-white"
  >
    <div
      class="max-w-4xl px-4 py-16 mx-auto sm:px-6 sm:py-24 lg:max-w-7xl lg:px-8 lg:flex lg:items-center lg:justify-between"
    >
      <h2
        class="text-4xl font-extrabold tracking-tight text-gray-900 sm:text-4xl"
      >
        <span
          class="block"
        >
          Ready to get started?
        </span>
        <span
          class="block text-transparent bg-gradient-to-r from-purple-600 to-indigo-600 bg-clip-text"
        >
          Get in touch or create an account.
        </span>
      </h2>
      <div
        class="mt-6 space-y-4 sm:space-y-0 sm:flex sm:space-x-5"
      >
        <a
          class="flex items-center justify-center px-4 py-3 text-base font-medium text-white border border-transparent bg-gradient-to-r from-purple-600 to-indigo-600 bg-origin-border rounded-md shadow-sm hover:from-purple-700 hover:to-indigo-700"
          href="#"
        >
          Learn more
        </a>
        <a
          class="flex items-center justify-center px-4 py-3 text-base font-medium text-indigo-800 border border-transparent rounded-md shadow-sm bg-indigo-50 hover:bg-indigo-100"
          href="#"
        >
          Get started
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
